staging/src/k8s.io/kubectl/pkg/drain/filters.go

```diff
@@ -256,3 +256,12 @@ func (d *Helper) skipDeletedFilter(pod corev1.Pod) PodDeleteStatus {
 	}
 	return MakePodDeleteStatusOkay()
 }
+
+func (d *Helper) isWorkloadDaemonSet(workloadRef *metav1.OwnerReference) bool {
```

Comment:
This doesn't need to be a method of Helper, it can be a normal function

Reply:
fixed